modules/platform/nuxeo-fsexporter/src/main/java/org/nuxeo/io/fsexporter/FSExporterPlugin.java

```diff
@@ -32,4 +32,9 @@
 
     File serialize(CoreSession session, DocumentModel docfrom, String fsPath) throws IOException;
 
+    /**
+     * @since 11.4
+     */
+    DocumentModelList getCollectedDocuments(CoreSession session, DocumentModel doc);
```

Comment:
This doesn't need to be a method in the `FSExporterPlugin` interface. Just inline the actual implementation into its caller, in `FSExporter.export` in the `if` statement.